pkg/config/matcher.go

```diff
@@ -28,3 +101,88 @@ func matchByAttributes(actual *processAttrs, required *Attributes) bool {
 	}
 	return true
 }
+
+func restartObjectWithAttributes(ctx context.Context, clientSet kubernetes.Interface, required *Attributes) error {
+	// pod labels is not compatible with metadata - unless it's namespace
```

Comment:
What does this mean? Do we mean that you can't define a criteria where you use both metadata attributes and pod labels?